index.html

```diff
@@ -250,6 +250,25 @@ <h2 class="supportive-text">
                 </div>
                 
             </section>
+
+              <!-- -----------------------------------------FOOTER CONTACT FORM ----------------------------------------------- -->
+      <footer class="contact-form">
+        <div class="contact-global-container">
+         <h2 class="contact-text-container">Contact me </h2>
+         <p class="contact-paragraph">If you have an application  you <br>are interested in developing, a <br> feature that you need built or a <br> project that needs coding.
+            I’d <br> love to help with it </p>
+            <form action="https://formspree.io/f/{form_id}" method="post" class="submit-form">
+             <input type="text" name="username" class="username" placeholder="Yeremias" minlength="30" required="required"> 
+             <input type="email" name="email" class="email" placeholder="natajayanj14@gmail.com" required="required">
+             <textarea name="text-field" class="text-area" cols="30" rows="10" minlength="500" placeholder="Write your message here" required="required"></textarea>
+             <button class="button-submit">Get It Touch</button>
+            </form>
+        </div>
+          
+       </div>
+      
+
+     </footer>
```

Comment:
- Please apply the required styling for your contact form. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your Project:
- Remember to use the `flex` display property to align content in this section 🙏 
<img width="280" alt="lllll" src="https://user-images.githubusercontent.com/73607512/207615821-4e0235d4-d215-4a4a-8c20-6434cb2c23e2.PNG">

Figma Design:
<img width="250" alt="qwec" src="https://user-images.githubusercontent.com/73607512/207616155-3533e910-8abe-4586-a240-8e88002a15bf.PNG">